content/about.md

```diff
@@ -2,11 +2,10 @@
 title: "About"
 ---
 
-Deis Labs is a shoutout to Deis, the organization where many of us started our
-journey into Kubernetes and Open Source - creating and nuturing the Helm project
+Deis Labs is a shoutout to [Deis](https://github.com/deis), the organization where many of us started our
+journey into Kubernetes and Open Source - creating and nuturing the [Helm](https://helm.sh/) project
```

Comment:
> nuturing

nurturing

Reply:
This is much better than sounding like we had Helm spayed

Reply:
Corrected, thanks @bridgetkromhout